flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/TableConfig.java

```diff
@@ -274,9 +263,10 @@ public void setMaxGeneratedCodeLength(Integer maxGeneratedCodeLength) {
 	 *
 	 * @param minTime The minimum time interval for which idle state is retained. Set to 0 (zero) to
 	 *                never clean-up the state.
-	 * @param maxTime The maximum time interval for which idle state is retained. Must be at least
-	 *                5 minutes greater than minTime. Set to 0 (zero) to never clean-up the state.
+	 * @param maxTime Currently maxTime will be ignored and it will automatically derived from minTime
```

Comment:
I would suggest to keep the original description for `maxTime`, but add a NOTE to explain the `maxTime` is ignored now and is inferred from `minTime` with a 1.5 multiplier.